qna/AT-carefree-extension-after-expired-grace-period.md

```diff
@@ -4,12 +4,12 @@ How to extend Carefree products which expired beyond the grace period?
 
 ## Metadata tags
 
-lang-en, market-at, bundle-carefree, portal, cashbox management, PosDealer
+lang-en, market-at, bundle-carefree, portal, PosOperator, PosDealer, Consultant
 
 ## Answer
 
-The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-months-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via portal, and a request for assistance will have to be placed with the fiskaltrust Support. 
+The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-months-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via portal, and a request for assistance will have to be placed with the fiskaltrust Support via eMail. 
```

Comment:
I don't see the need for specifying that this should go via email. Fiskaltrust has defined support channels and such requests can go via any of them. We are going to change email-based support to a form based one in some near future and then we'd have to update all documents. Let's remove this part.

Reply:
changed